c2corg_ui/static/js/trackdownload.js

```diff
@@ -74,7 +74,8 @@ app.TrackDownloadController.prototype.downloadFeatures_ = function(
 /**
  * @export
  */
-app.TrackDownloadController.prototype.downloadGpx = function() {
+app.TrackDownloadController.prototype.downloadGpx = function($event) {
```

Comment:
The JS doc of the function should be updated to make this new argument appear.
Perhaps
```
* @param {goog.events.Event | jQuery.Event} $event
```
See for instance https://github.com/c2corg/v6_ui/blob/5cd68ec0a539e281b4fe5aac27b6a07ce6cd8370/c2corg_ui/static/js/viewdetails.js#L445-L453

You can also use a more generic type like
```
* @param {Object} $event
```
but that's not a good practise to use too generic types if you can use a more precise one.

I would also recommend to name the argument ``event`` instead of ``$event`` in the function definition (as for other methods with an event argument).